app/assets/javascripts/student_profile_v2/attendance_details.js

```diff
@@ -7,6 +7,31 @@
   var ProfileChart = window.shared.ProfileChart;
   var Scales = window.shared.Scales;
 
+  var styles = {
+    box: {
+      border: '1px solid #eee',
+      padding:'15px',
+      marginTop: '10px',
+      marginBottom: '10px',
+      width: '50%'
+    },
+    item: {
+      width: '33%',
```

Comment:
For layout styling like this, where we want three items to have equal width and take up the full width of the container, could we use [flexbox](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Using_CSS_flexible_boxes)?